### ISSUE TITLE
Field Names for Creating an Organization

### ISSUE THREAD

**Issue by CarlyNel** [terraframe/geoprism-registry#337]
Instance: 
- Staging - Firefox
- 0.8.1 on 22 June 2020

As an SRA I was trying to create an organization.
When the pop up window opened for me to complete, I found the fields a bit confusing (see screenshot).

I expected the field names to be a bit clearer in terms of what information I was expected to fill in. I also found the “defaultLocale” confusing but I see that there is already a [ticket](https://app.zenhub.com/workspaces/geoprism-registry-5df81d4b734ff217d40b12a4/issues/terraframe/geoprism-registry/173) for this.

**Low Priority**

![image.png](https://images.zenhubusercontent.com/5eda94a201596292f642f925/8f5623d8-bd93-418c-9e8f-bdff3ea07ef0)

**Comment by justinlewis** [comment]
@CarlyNel Moved to in review because you linked to this from another ticket I worked on. Hoping to review this in testing.

**Comment by CarlyNel** [comment]
Hey @justinlewis. I'm logged in as a sys admin but am seeing exactly the same as what I saw originally (see screenshot below). What was changed?

My recommendation was to change "Name" to read as "Organization name" and "contact" to "contact person/number" (whatever this field is referring to). It also still says 'default locale' -> not sure what purpose this is serving or what it's trying to convey.

![image.png](https://images.zenhubusercontent.com/5eda94a201596292f642f925/db01a8f4-9f05-4a6a-9a2a-657646cc569c)
